plugins/protocolbuffers/csharp/v26.1/buf.plugin.yaml

```diff
@@ -7,3 +7,13 @@ output_languages:
   - csharp
 spdx_license_id: BSD-3-Clause
 license_url: https://github.com/protocolbuffers/protobuf/blob/v26.1/LICENSE
+registry:
```

Comment:
The grpc/csharp plugin depends on v26.1 so I went ahead and backfilled this config to versions v26.1 through v27.1 as well.